src/components/SearchField/SearchField.js

```diff
@@ -12,7 +12,7 @@ import "./SearchField.scss";
 
 export function SearchField({ instant }) {
   const [value, setValue] = useState("");
-  const [isError, setIsError] = useState(false);
+  const [setIsError] = useState(false);
```

Comment:
in this case `setIsError` - is not a setter. why did you change it?

Reply:
i will return it, it is my mistake